lib/modules/versioning/oci-helm/index.ts

```diff
@@ -0,0 +1,87 @@
+import semver from 'semver';
+import type { RangeStrategy } from '../../../types/versioning';
+import { regEx } from '../../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../generic';
+import type { NewValueConfig, VersioningApi } from '../types';
+
+export const id = 'oci-helm';
+export const displayName = 'OCI Helm';
+export const urls = [
+  'https://semver.org/',
+  'https://helm.sh/docs/chart_best_practices/dependencies/#versions',
+  'https://github.com/Masterminds/semver#basic-comparisons',
+  'https://helm.sh/docs/topics/registries/#oci-feature-deprecation-and-behavior-changes-with-v380',
+];
+export const supportsRanges = true;
+export const supportedRangeStrategies: RangeStrategy[] = [
```

Comment:
I don't see those implemented in getNewValue

Reply:
Initially, I had the idea to re-implement https://github.com/Masterminds/semver to support every possible Helm version range variation. But as mentioned in https://github.com/renovatebot/renovate/pull/25029#issuecomment-1808608317, I decided to go with composer as the most fitting range versioning in this PR. I was short on time that's why it's not yet adjusted. Working on it right now. 🙂

Reply:
Ok, I dropped the range support for now. 🙂